tests/trainer/test_dataloaders.py

```diff
@@ -1159,3 +1159,71 @@ def test_replace_sampler_with_multiprocessing_context(tmpdir):
 
     new_data_loader = trainer.replace_sampler(train, SequentialSampler(train.dataset))
     assert (new_data_loader.multiprocessing_context == train.multiprocessing_context)
+
```

Comment:
Could we add a test of Flash's DataPipeline Mock for the PR?

Reply:
Yes. Good idea !